shared/editor/embeds/Grist.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import Frame from "../components/Frame";
+import { EmbedProps as Props } from ".";
+
+const URL_REGEX = new RegExp(
+  "^https?://([a-z.-]+\\.)?getgrist\\.com/(.*)/p/([0-9]+)\\?embed=true$"
+);
+
+export default class Grist extends React.Component<Props> {
```

Comment:
Apologies for intermittent and disruptive reviews. It's so because I'm currently on a beginner's track and just getting familiar with the practices followed within the project.

One of the practices is that the newly developed components should preferably be functional instead of class-based. Therefor, you might want to change this. You may use [`Gliffy.tsx`](https://github.com/outline/outline/blob/74d9409cc32c5923fe72e98eb961d1d56304519e/shared/editor/embeds/Gliffy.tsx) as reference.

Reply:
Ok, Ill try and revisit it tonight or tomorrow

Reply:
Ok, new version committed now